support-redemptiondb/build.sbt

```diff
@@ -3,7 +3,7 @@ name := "support-redemptiondb"
 description:= "packages the database cfn for the redemption code db"
 
 assemblyJarName := "dummy.jar"
-riffRaffPackageType := assembly.value
+riffRaffPackageType := new File("dummy.jar")
```

Comment:
I changed this so we didn't need the sbt-assembly pluigin enabled on this project.